extensions/wikia/LookupUser/LookupUser.body.php

```diff
@@ -518,4 +522,35 @@ public static function isUserFounder( $userName, $wikiId ) {
 	private static function getFounderMemKey( $userName, $wikiId ) {
 		return wfSharedMemcKey( 'lookupUser', 'isUserFounder', $userName, $wikiId );
 	}
+
+	public static function fetchDiscussionPostCountAndDate( $username, $siteId ) {
+		$wg = F::app()->wg;
+
+		$targetUser = User::newFromName($username);
+		if (WikiFactory::getVarValueByName( 'wgEnableDiscussions', $siteId )) {
+			$posts = self::getDiscussionContributionApi()->getPosts($siteId, $targetUser->getId());
+			$count = $posts->getPostCount();
+			if ($count > 0) {
+				$halForumThreadEmbeddeds = $posts->getEmbedded();
+				$dsf = $halForumThreadEmbeddeds->getDocposts()[0]->getCreationDate()->getEpochSecond();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$dsf" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)